companies/wallwisher-inc.json

```diff
@@ -0,0 +1,17 @@
+{
+    "slug": "wallwisher-inc",
+    "relevant-countries": [
+        "all"
+    ],
+    "name": "Wallwisher, Inc.",
+    "runs": [
+        "padlet.com"
+    ],
+    "address": "981 Mission St\nSan Francisco, CA 94103\nUSA",
```

Comment:
Just a nitpick, change to `United States of America` since that's what we're using on all of them for consistency